backend/src/schema/resolvers/searches.spec.js

```diff
@@ -29,7 +29,7 @@ afterAll(async () => {
 
 const searchQuery = gql`
   query($query: String!) {
-    findResources(query: $query, limit: 5) {
+    searchResults(query: $query, limit: 5) {
```

Comment:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-05-13T17:49:04Z" title="Wednesday, May 13th 2020, 7:49:04 pm +02:00">May 13, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -24,12 +24,12 @@ beforeAll(async () => {
 })
 
 afterAll(async () => {
-  await cleanDatabase()
+  //  await cleanDatabase()
```


Comment must be removed.